js/slider/jquery.slider.js

```diff
@@ -182,23 +182,28 @@ jQuery( function($){
 
 				// Set up showing and hiding navs
 				$p.add($n).hide();
-				if( !$base.hasClass('sow-slider-is-mobile') && $slides.length > 1 ) {
-
-					var toHide = false;
-					$base
-						.mouseenter(function(){
-							$p.add($n).clearQueue().fadeIn(150);
-							toHide = false;
-						})
-						.mouseleave(function(){
-							toHide = true;
-							setTimeout(function(){
-								if( toHide ) {
-									$p.add($n).clearQueue().fadeOut(150);
-								}
+				if( $slides.length > 1 ) {
+					if( !$base.hasClass('sow-slider-is-mobile') ) {
+
+						var toHide = false;
+						$base
+							.mouseenter(function(){
+								$p.add($n).clearQueue().fadeIn(150);
 								toHide = false;
-							}, 750);
-						});
+							})
+							.mouseleave(function(){
+								toHide = true;
+								setTimeout(function(){
+									if( toHide ) {
+										$p.add($n).clearQueue().fadeOut(150);
+									}
+									toHide = false;
+								}, 750);
+							});
+					} else if ( settings.nav_always_show_mobile && window.matchMedia('(max-width: ' + settings.breakpoint + ')').matches) {
```

Comment:
Hmmm... so it seems I forgot that the Slider is already detecting mobile devices, by using `wp_is_mobile` on the back end. Does it makes to use both that and then have a breakpoint too? What if we have mobile devices which have more than 780px width?

Reply:
Ideally, `wp_is_mobile` shouldn't ever be used as it won't work when the user is using a cache plugin (and that's _very_ common). We should ideally remove `wp_is_mobile` in favor of using the breakpoint.

Reply:
Ah, that's no good. 😕 My issue here is that this is trying to detect whether or not a device uses touch input, by screen size. For example, this won't work on a touch screen device with >780px width. It seems there's no reliable method for this though, so breakpoint will have to do. :finnadie:

Reply:
Do you want to move away from `wp_is_mobile` or stick with it for the time being? It's not ideal for users using caching, but we haven't received many reports.

Reply:
Yeah, rather don't change what we're already using for now. I think we can relook at this later if it becomes a big issue (gets a lot of support requests).